tasks/post_copy_request_to_queue/requirements-dev.txt

```diff
@@ -1,7 +1,13 @@
-boto3~=1.12.47
-moto[sqs]
-pytest==6.1.2
+## Testing libraries
 coverage==5.3
-cumulus-message-adapter-python>=1.1.1
-SQLAlchemy~=1.4.13
-psycopg2-binary==2.8.6
\ No newline at end of file
+pytest==6.1.2
+
+## Libraries needed for testing the application
+boto3==1.12.47
```

Comment:
Wouldn't boto3 also be an Application Library?

Reply:
boto3 is not imported or used in the `post_copy_request_to_queue.py` file but it is imported and used in the `test_post_copy_request_to_queue.py` file to perform some testing validation against unit tests and items being sent to fake SQS queues. That is why it is neded for application testing currently but not as part of the application itself.